packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -609,4 +600,30 @@ export class KeytipLayerBase extends BaseComponent<IKeytipLayerProps, IKeytipLay
     this.setState({ inKeytipMode: inKeytipMode });
     this._keytipManager.inKeytipMode = inKeytipMode;
   };
+
+  /**
+   * Emits a warning if duplicate keytips are found for the current keytip
+   */
+  private _detectDuplicateKeytips = (): void => {
+    const duplicateKeytips = this._getDuplicateIds(this._keytipTree.getChildren());
+    if (duplicateKeytips.length) {
+      warn('Duplicate keytips found for ' + duplicateKeytips.join(', '));
```

Comment:
Would it be interesting/useful to know the full sequence to get to where the duplicate keytips were found (to know "where in the tree" this is getting logged from)?

Reply:
This will log the full sequence, in a sense. It will log 'ktp-a-b-c' which means the path to it is A, B, C